language/polkavm/examples/basic/sources/multiple_functions.move

```diff
@@ -0,0 +1,47 @@
+module 0xdead::functions {
```

Comment:
Tiny Nit. I would pick something invisible like 0x7612 that doesn't stand out. But that is just my 2-cents